galite-tests/src/test/kotlin/org/kopi/galite/tests/form/VBlockTests.kt

```diff
@@ -91,21 +91,17 @@ class VBlockTests : JApplicationTestBase() {
   }
 
   @Test
-  @Ignore("TODO: this doesn't seem to be right")
   fun selectLookupTest() {
```

Comment:
The name of this test should be renamed as it represents a fail scenario.
We should have another test that expects selectLookup to work successfully.